modules/core/src/main/scala/org/aossie/agora/model/Ballot.scala

```diff
@@ -21,7 +21,7 @@ case class PreferenceBallot(
 
 }
 
-case class ScoreBallot(scores: List[(Candidate, Rational)], override val id: Int, w: Rational)
+class ScoreBallot(val scores: List[(Candidate, Rational)], override val id: Int, w: Rational)
```

Comment:
Follow same recommendation as stated for PrefenceBallot here. Add type parameter for Candidate say C and revert to case class.